Classes/Issues/Comments/Tables/IssueCommentTableModel.swift

```diff
@@ -31,12 +31,21 @@ final class IssueCommentTableModel: NSObject, ListDiffable {
         }
     }
 
-    let rowHeight = Styles.Sizes.labelEventHeight
+    let rowHeight: CGFloat
     let columns: [Column]
     let totalHeight: CGFloat
 
     init(columns: [Column]) {
         self.columns = columns
+        self.rowHeight = columns.reduce(0) { currentRowHeight, column  in
+            let maxHeight: CGFloat = column.rows.reduce(0.0) { currentMaxHeight, row in
+                let textSize = row.text.computeSize(0)
+                return max(currentMaxHeight, textSize.height)
+            }
+            
+            let inset = IssueCommentTableCollectionCell.inset.top + IssueCommentTableCollectionCell.inset.bottom
+            return max(currentRowHeight, maxHeight + inset)
+        }
```

Comment:
I'm going to strip this part out and revisit. This is an n^2 algo that should be calculated at the parsing phase. Asking the text sizing for the computeSize again might end up doing wasted work.

Reply:
Makes sense, have to say I didn't like it that much myself but fixed the problem 🤗 Parsing phase is happening MMarkdown, I presume?